.github/workflows/test.yml

```diff
@@ -22,14 +23,28 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
           cache: npm
-      - run: npm ci --ignore-scripts
-      - run: npm run build
-      - run: npm run test:esm
-      - run: npm run test:features
-      - run: npm run test:qunit
-      - run: npm run test:no:specs
-      - run: npm run test:cjs
-      - run: npm run lint:ci
+      - name: Install dependencies
+        run: npm ci --ignore-scripts
+      - name: Build
+        run: npm run build
+      - name: "Test: ESM"
```

Comment:
the names are optional... it just uses the command in the action output/logs, but doesnt matter

Reply:
Yeah just makes it look a little nicer, if you have objections let me know.